packages/seacas/libraries/ioss/src/Ioss_ChangeSet.C

```diff
@@ -43,17 +43,39 @@ int file_exists( const Ioss::ParallelUtils &util,
   return file.parallel_exists(util.communicator(), message);
 }
 
-bool internal_decomp_specified(const Ioss::PropertyManager& props)
+std::string get_decomposition_property(const Ioss::PropertyManager &properties,
```

Comment:
That looks good.  At some point we maybe move to the Ioss::Utils class to avoid the code dupication, but good for now.